src/components/UI/navbar/index.js

```diff
@@ -47,21 +47,23 @@ const Navbar = () => {
             setIsLoggedIn(false);
             throw new Error(`${response.status} (${response.statusText})`);
           }
+
           return response.json();
         })
         .then((responseJson) => {
-          if (!responseJson.incompleteUserDetails) {
-            setIsLoggedIn(true);
-            setUserData({
-              userName: responseJson.username,
-              firstName: responseJson.first_name,
-              profilePicture: responseJson.picture.url?.DEFAULT_AVATAR,
-            });
+          if (responseJson.incompleteUserDetails) {
+            window.open('https://my.realdevsquad.com/signup', '_blank');
           }
-          return window.open('https://my.realdevsquad.com/signup');
+
+          setIsLoggedIn(true);
+          setUserData({
+            userName: responseJson.username,
+            firstName: responseJson.first_name,
+            profilePicture: responseJson.picture?.url ?? DEFAULT_AVATAR,
+          });
         })
         .catch((err) => {
-          return err;
+          console.error(err);
```

Comment:
Was it decided to go ahead with `console.error` here?

Reply:
Let's leave it for now.